common/static/common/templates/discussion/forum-actions.underscore

```diff
@@ -3,10 +3,10 @@
         <% _.each(primaryActions, function(action) { print(_.template($('#forum-action-' + action).html())({})) }) %>
         <li class="actions-item is-visible">
             <div class="more-wrapper">
-                <a href="javascript:void(0)" class="action-button action-more" role="button" aria-haspopup="true" aria-controls="action-menu-<%= contentId %>">
-                    <span class="action-label"><%- gettext("More") %></span>
+                <button class="btn-link action-button action-more" aria-label="<%- gettext('More') %>" aria-haspopup="true" aria-controls="action-menu-<%= contentType %>">
```

Comment:
I was getting an error from the a11y acceptance tests without it. Any ideas why that would be?